lesson-01-cw721/exercise-04/exercise.md

```diff
@@ -15,7 +15,7 @@ Let's think about what the use cases are for importing the `cw721` package into
 
 ### When to import cw721
 
-- Querying an NFT collection from inside a contract and enforcing the correct response type
+- Querying an NFT collection from inside a contract and enforcing the correct response types
```

Comment:
I actually think we need to use `type` here (otherwise, we'd have to change `Querying an NFT collection...` to `Making queries to an NFT collection...`)

When you make a query only one response type is returned.